Objects/methodobject.c

```diff
@@ -160,7 +160,8 @@ PyCMethod_GetClass(PyObject *op)
 static void
 meth_dealloc(PyCFunctionObject *m)
 {
-    _PyObject_GC_UNTRACK(m);
+    PyObject_GC_UnTrack(m);
+    Py_TRASHCAN_SAFE_BEGIN(m);
```

Comment:
Use the new trashcan macros. These are broken: https://bugs.python.org/issue40608